src/js/components/Tag/Tag.js

```diff
@@ -9,7 +9,10 @@ import { StyledRemoveButton, StyledTagButton } from './StyledTag';
 import { useThemeValue } from '../../utils/useThemeValue';
 
 const Tag = forwardRef(
-  ({ name, value, size = 'medium', onRemove, onClick, ...rest }, ref) => {
+  (
+    { name, value, size = 'medium', onRemove, onClick, ariaLabel, ...rest },
```

Comment:
This approach seems like it would apply to the tag, not the subcomponent. Wondering if we should support

```
removeProps= { 'aria-label': '' }
```

For now I'd say we only let aria-label be the prop in there.

Reply:
I would agree with this and in fact given how `rest` is put on the container you'd potentially break existing usage of ariaLabel by apps.